File: platforms/ios/Dezel/Dezel/View/TextAreaObserver.swift
/**
 * @protocol TextAreaObserver
 * @since 0.7.0
 */
public protocol TextAreaObserver: AnyObject {

	/**
	 * @method didChange
	 * @since 0.7.0
	 */
	func didChange(textInput: TextArea, value: String)

	/**
	 * @method didFocus
	 * @since 0.7.0
	 */
	func didFocus(textInput: TextArea)

	/**
	 * @method didBlur
	 * @since 0.7.0
	 */
	func didBlur(textInput: TextArea)

}

File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/view/TextAreaObserver.kt
package ca.logaritm.dezel.view

/**
 * @interface TextAreaObserver
 * @since 0.7.0
 */
public interface TextAreaObserver {

	/**
	 * @method onChange
	 * @since 0.7.0
	 */
	fun onChange(textArea: TextArea, value: String)

	/**
	 * @method onFocus
	 * @since 0.7.0
	 */
	fun onFocus(textArea: TextArea)

	/**
	 * @method onBlur
	 * @since 0.7.0
	 */
	fun onBlur(textArea: TextArea)

}
